.github/workflows/publish-docker-image.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish Docker image
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+
+      - name: Log in to Docker Hub
+        uses: docker/login-action@f054a8b539a109f9f41c372932f1ae047eff08c9
```

Comment:
Any specific reason for the commit-hashes instead of the version (in this example v2)?

Reply:
No, lol, this is copied from another docker setup I use. Let me get the pinned versions real quick.